ewitem.py

```diff
@@ -962,7 +962,7 @@ def find_item(item_search = None, id_user = None, id_server = None):
 				item = item_sought.get('name')
 			)
 
-			if item_sought.get('id_item') == user_data.weapon:
+			if item_sought.get('id_item') == int(user_data.weapon):
```

Comment:
why is the int cast necessary? that's gonna cause problems, if the user doesn't have a weapon equipped (int(""))

Reply:
that's cause weapon column is a varchar, so comparing the item's id and the weapon is always false

Reply:
oh right. we should just make it an int column then